direct/core/src/test/java/cz/o2/proxima/direct/storage/InMemStorage.java

```diff
@@ -169,15 +180,21 @@ public static long getIdleFlushTime() {
 
   public final class Writer extends AbstractOnlineAttributeWriter {
 
-    private Writer(EntityDescriptor entityDesc, URI uri) {
+    private final int numPartitions;
+    private final Partitioner partitioner;
+
+    private Writer(
+        EntityDescriptor entityDesc, URI uri, int numPartitions, Partitioner partitioner) {
       super(entityDesc, uri);
+      this.numPartitions = numPartitions;
+      this.partitioner = partitioner;
     }
 
     @Override
     public void write(StreamElement data, CommitCallback statusCallback) {
       NavigableMap<Integer, InMemIngestWriter> writeObservers = getObservers(getUri());
       final ArrayList<InMemIngestWriter> currentWriters;
-      try (Locker lock = holder().lockWrite()) {
+      try (Locker ignore = holder().lockWrite()) {
```

Comment:
How does that improve clarity? :-)

Reply:
IDE / static analysis does not highlight it as unused